### DISCUSSION TITLE
Pag     Unable to display on the webpage

### DISCUSSION THREAD

**Discussion by Momo356** [Tencent/libpag#1864] [Q&A] [answered]
Ae exported the pag file without any issues, and also imported it into DW according to the steps on the official website. However, the resulting code was not displayed on the webpage after writing, and I was unable to download the sample on the official website

**Comment by Momo356** [comment]
就是Ae导出的Pag文件没有问题，然后按照官网的那个步骤一步步进行，导入到代码的软件，在网页没有出现应该有的动画效果。然后我去官网下了demo，然后我的电脑也没有出现官网的那个例子。

**Reply by zhalice2011** [reply, marked as answer]
Here are some pag demo codes:  https://github.com/libpag/pag-web

Here is the online sample code: https://pagtest.zhalice.com/pages/simple.html

If it still cannot run, please upload your AE project or PAG file and demo example.
